crates/turbopack-ecmascript/tests/tree-shaker/analyzer/1/output.md

```diff
@@ -709,6 +743,14 @@ import { foobar } from "__TURBOPACK_PART__" assert {
 };
 foobar += "foo";
 
+```
+## Part 8
+```js
+export { foobar };
+export { foo };
+export { external1 };
+export { external2 };
+
```

Comment:
That doesn't look right since it exports undeclared symbols... Should it have some imports too?

Reply:
This entry was `Key::Exports`. My bad, I missed it.